app/src/main/java/com/jwisozk/flightreviews/ButtonHolder.kt

```diff
@@ -0,0 +1,21 @@
+package com.jwisozk.flightreviews
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.Button
+import androidx.annotation.NonNull
+
+class ButtonHolder(@NonNull inflater: LayoutInflater, @NonNull parent: ViewGroup) :
+    AbsHolder(inflater, parent, R.layout.submit_button) {
+
+    override fun bind(paramCell: AbsParamCell) {
+        if (paramCell.viewType != AbsParamCell.ViewType.SUBMIT_BUTTON)
+            return
+        val buttonResultCell = paramCell as ButtonParamCell
+        val button = itemView.findViewById<Button>(R.id.submitButton)
```

Comment:
Also note that we don't have to find our views by id in the `bind` method, we can do it once making it a `private val ...` class-level field